topics/web/apis/example3/main.go

```diff
@@ -24,7 +24,8 @@ func App() http.Handler {
 	r.Get("/customers/{id}", showHandler)
 	r.Get("/customers", indexHandler)
 	r.Post("/customers", createHandler)
-	r.Get("/", indexHandler)
+	// Redirect requests to / to /customers
```

Comment:
If you want this comment here are the rules.

A comment can't have anything on top of it.
A comment must have proper sentence structure and punctuation.